pkg-manager/core/test/install/overrides.ts

```diff
@@ -2,15 +2,13 @@ import path from 'path'
 import fs from 'fs'
 import { sync as readYamlFile } from 'read-yaml-file'
 import { PnpmError } from '@pnpm/error'
-import { prepareEmpty, preparePackages } from '@pnpm/prepare'
+import { prepare, prepareEmpty, preparePackages } from '@pnpm/prepare'
 import { addDistTag } from '@pnpm/registry-mock'
 import { WANTED_LOCKFILE } from '@pnpm/constants'
 import { type MutatedProject, type ProjectOptions, addDependenciesToPackage, mutateModulesInSingleProject, mutateModules } from '@pnpm/core'
 import { type LockfileFileV9 } from '@pnpm/lockfile.types'
 import { type ProjectRootDir, type ProjectManifest } from '@pnpm/types'
-import {
-  testDefaults,
-} from '../utils'
+import { testDefaults } from '../utils'
```

Comment:
This 3 lines look kinda ugly, so I changed it to 1. The change is too small, not worth opening a separate PR, so I just include it here.